siliconcompiler/server.py

```diff
@@ -156,8 +159,8 @@ async def handle_remote_run(self, request):
         # Create a dummy Chip object to make schema traversal easier.
         # TODO: if this is a dummy Chip we should be able to use Schema class,
         # but looks like it relies on chip.status.
-        chip = Chip(cfg['design']['value'])
-        chip.schema.cfg = cfg
+        chip = Chip('server')  # start with a dummy name, as this will be overwritten
+        chip.schema = Schema(cfg=cfg) # Add provided schema
```

Comment:
This is a very minor comment, but I'm not sure about putting comments at the end of lines in our public code. It's nice for brief examples and documentation, but maybe move these comments to the line above?